app.js

```diff
@@ -14,11 +14,8 @@ app.get('/api', getAllApi)
 
 app.get('/api/articles/:article_id', getArticleById)
 
-app.all('*', (err, res) => {
-  console.log("going through middleware");
-  res.status(404).send({msg: 'Not Found' });
-});
-
+app.all('*', errorHandlerOne);
+app.use('/api/articles/notAnId', errorHandlerTwo)
```

Comment:
I don't think you need to specify a path for this error handling middleware, what part of your code do you think is triggering this error handling function?

Hint: Its nothing we're doing in this file.

Reply:
the catch in controller?